src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -147,19 +165,19 @@ public GHTeam getTeamByName(String name) throws IOException {
 
     /**
      * Finds a team that has the given slug in its {@link GHTeam#getSlug()}
-     *
+     * 
      * @param slug
      *            the slug
      * @return the team by slug
      * @throws IOException
      *             the io exception
+     * @see <a href= "https://developer.github.com/v3/teams/#get-team-by-name">documentation</a>
      */
     public GHTeam getTeamBySlug(String slug) throws IOException {
-        for (GHTeam t : listTeams()) {
-            if (t.getSlug().equals(slug))
-                return t;
-        }
-        return null;
+        return root.createRequest()
```

Comment:
This is great!  But you'll need to update the test data for this test.  See CONTRIBUTING.md.